<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginModeButtons</class>
 <widget class="QGroupBox" name="CLoginModeButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login mode </string>
  </property>
  <property name="title">
   <string/>
  </property>
  <property name="flat">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="hl_LoginMode">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_LognMode">
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb_LoginNormal">
     <property name="text">
      <string>Normal</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb_LoginStealth">
     <property name="text">
      <string>Stealth</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb_LoginObserver">
     <property name="text">
      <string>Observer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
